aperturedb/ParallelLoader.py

```diff
@@ -10,12 +10,14 @@
 
 
 class ParallelLoader(ParallelQuery.ParallelQuery):
-    """**Parallel and Batch Loader for ApertureDB**"""
+    """
+    **Parallel and Batch Loader for ApertureDB**
 
-    def __init__(self, db, dry_run=False):
 
-        super().__init__(db, dry_run=dry_run)
+    """
 
+    def __init__(self, db, dry_run=False, response_handler=None):
```

Comment:
we no longer need response_handler=None on the constructor, right? Or the idea is to keep it there?

Reply:
mmm but why keep it there if we don't plan to use it? Only the response_handler on the generator will be used.

Reply:
will remove.